### DISCUSSION TITLE
请问有办法实现点击billbill链接跳转uwp吗

### DISCUSSION THREAD

**Discussion by cc1477** [ywmoyue/biliuwp-lite#482] [feature]
### 问题描述

通过点击视频链接跳转到uwp播放

### 描述解决方案

无

### 代码贡献

否

**Comment by ywmoyue** [comment]
本来就支持bilibili协议的链接，如果想要点击http的bv号链接跳转到uwp的话，可能得写个油猴脚本转一下
